src/util.js

```diff
@@ -162,8 +162,8 @@ export function sshonify(results, description, requestParams, truncated) {
     ':version': 3,
     results: {
       limit: Math.max(requestParams.limit || 1, results.length),
-      offset: requestParams.offset || 0,
-      total: requestParams.offset || 0 + results.length + (truncated ? 1 : 0),
+      offset: parseInt(requestParams.offset, 10) || 0,
```

Comment:
```suggestion
      offset: isNaN(requestParams?.offset) ? 0 : parseInt(requestParams.offset, 10)
```

Reply:
Potential logic issue with the suggested change if offset is not provided, and I do not find the code more readable.  Will skip this one but I did commit the other suggestion via ci5650.

Reply:
@langswei Fair enough; mine is more verbose, both give the same outcome, I assume shorter is better there.